developing/docker/README.md

```diff
@@ -87,3 +87,26 @@ unused networks:
 See [the docker system prune
 documentation](https://docs.docker.com/engine/reference/commandline/system_prune/)
 for more options like pruning volumes
+
+## Docker Configuration
+
+You can configure all sorts of defaults in the [docker
+config.json](https://docs.docker.com/engine/reference/commandline/cli/#configuration-files).
+
+### Keybindings
+
+If you dislike the default detach keybindings of `CTRL-p CTRL-q`, you
+can override it in `config.json` with something like:
+
+    {
+      "detachKeys": "ctrl-@,ctrl-["
+    }
+
+### Credentials Store
+
+You can store docker credentials in an [external credentials
+store](https://docs.docker.com/engine/reference/commandline/login/#credentials-store).
+
+On macOS, the keychain will be used by default, but if you need/want
+to store shared credentials, [pass](https://www.passwordstore.org/)
```

Comment:
is this a recommended tool over aws vault, 1password, or other tools someone might be using to store secrets?

if so, it should probably be documented at a higher level, not necessarily in the docker page here.

Reply:
No recommendation intended, just pointing out capabilities of docker that could prove useful depending on the problem you are trying to solve.

Reply:
1password has a commandline utility that can be installed with `brew install 1password-cli`. Can we suggest that instead, to keep everyone on the same tool?

Reply:
Sure!  Please update the PR to document how that works with docker.  I'm just passing along information already included in the docker documentation at the link I provided.